src/nvim/main.c

```diff
@@ -188,7 +188,7 @@ bool event_teardown(void)
 /// Performs early initialization.
 ///
 /// Needed for unit tests. Must be called after `time_init()`.
-static void early_init(mparm_T *paramp)
+void early_init(bool clean_arg)
```

Comment:
why remove static?

Reply:
I've added it myself (IIRC), but it does not work with the unittests then.
Check https://github.com/neovim/neovim/pull/11768/files#diff-21cf87db76bbcab0a3b5054f99b4ef85L190 for the actual diff.